codegen-server-test/build.gradle.kts

```diff
@@ -13,6 +13,7 @@ plugins {
     java
     alias(libs.plugins.smithy.gradle.base)
     alias(libs.plugins.smithy.gradle.jar)
+    id("com.dorongold.task-tree") version "2.1.1"
```

Comment:
should be consistent with how we define dependencies by placing them in the plugins section of libs.versions.toml